app/(app)/zone-photo.tsx

```diff
@@ -20,9 +25,22 @@ const images = [
   'https://via.placeholder.com/150/F0E68C',
 ]
 
-const AppRoute = () => {
+const ZonePhotoPage = () => {
+  const { data, isPending, isError, error } = useQueryWithFocusRefetch(getFavoritePhotosOptions())
   const setState = useSetOffenceState()
 
+  if (isPending) {
+    return <LoadingScreen title="Vyberte zónovú značku" asScreenView />
+  }
+  console.log(data)
+  if (isError) {
+    return <Typography>Error: {error.message}</Typography>
+  }
+
+  if (data?.photos.length === 0) {
```

Comment:
If done in right, `data` should be defined after checking `isPending` and `isError`

https://tanstack.com/query/latest/docs/framework/react/guides/queries

>For most queries, it's usually sufficient to check for the isPending state, then the isError state, then finally, assume that the data is available and render the successful state